src/main/java/org/jboss/xavier/sanitizer/XavierPayloadSanitizer.java

```diff
@@ -0,0 +1,169 @@
+package org.jboss.xavier.sanitizer;
+
+import com.jayway.jsonpath.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class XavierPayloadSanitizer
+{
+    private static String inputFileName;
+    private static String outputFileName;
+    private static String issuesConditionsJSONFileName;
+
+    public static void main(final String[] args)
+    {
+        List<String> argsList = Arrays.asList(args);
+        argsList.forEach(arg -> {
+            if( arg.equals("--input"))
+            {
+                inputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+            if( arg.equals("--output"))
+            {
+                outputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+
+            if( arg.equals("--issues"))
+            {
+                issuesConditionsJSONFileName = argsList.get(argsList.indexOf(arg) + 1);
```

Comment:
Do you think it is worth checking that `issuesConditionsJSONFileName` is a JSON file? All the logic is based in JSON Path Queries so if `issuesConditionsJSONFileName` is not a JSON file, then everything will fail.

Reply:
issuesConditionsJSONFilename isn't a json file -  as you've commented below it needs renaming to something more appropriate. I assume you mean it's worth checking the inputFile is json formatted. The best I think we can do is catch the invalid json exception thrown by the parse method and provide a more user-friendly error message to the user.